### PR TITLE
chore: add dry run InfracostComment in example tests

### PR DESCRIPTION
**Pull request by hugorut**
* calls InfracostComment with dryRun flag to test `infracost output` and api interactions are sound

### CODE REVIEW

File: examples/multi-project/README.md [hunk 1]
```diff
@@ -38,6 +38,7 @@ jobs:
       - task: InfracostComment@0
         displayName: Post the comment
         inputs:
+          githubToken: $(githubToken)
```
Comment (aliscott) [inline examples/multi-project/README.md:41]: 🙌

File: .azure/pipelines/examples-test.yml [hunk 2]
```diff
@@ -1,5 +1,6 @@
 pr:
   - master
+  - azure-devops-tasks
```
Comment (hugorut) [inline .azure/pipelines/examples-test.yml:3]: will remove this once we've merged into master

File: tasks/InfracostComment/index.ts [hunk 3]
```diff
@@ -150,6 +150,10 @@ function setupRepoProvider(): RepoProvider {
 
   setEnvVariable(provider.envVariable, token, true);
 
+  if (taskLib.getVariable('System.Debug') === 'true') {
+    setEnvVariable("DEBUG", "*")
```
Comment (aliscott) [inline tasks/InfracostComment/index.ts:154]: 🚀

**Review by aliscott — APPROVED** [review]
LGTM!

**Pull request merged**